CHANGELOG_additions.md

```diff
@@ -0,0 +1,118 @@
+* Fixes flaky ScalablePushBandwidthThrottleIntegrationTest caused by timeout ([#8410](https://github.com/confluentinc/ksql/pull/8410)) ([fbbc076](https://github.com/confluentinc/ksql/commit/fbbc076a9e2471744ae47e96511ea4cf09c52540))
+* make to_bytes work with lowercase hex ([#8423](https://github.com/confluentinc/ksql/pull/8423)) ([fee904e](https://github.com/confluentinc/ksql/commit/fee904e07802195cd64432454d5e05dd6fb87099))
+* tolerate out-of-order execution on HARouting tests ([#8415](https://github.com/confluentinc/ksql/pull/8415)) ([c4cc6d5](https://github.com/confluentinc/ksql/commit/c4cc6d5c58dcbc82d2259192cea70eb2ae65ffba))
+* fix warning ([#8397](https://github.com/confluentinc/ksql/pull/8397)) ([66c96f4](https://github.com/confluentinc/ksql/commit/66c96f4424f97464d460851ba4dc767f36c24c25))
+* synchronizing writes to localcommand file ([#8406](https://github.com/confluentinc/ksql/pull/8406)) ([ed50b05](https://github.com/confluentinc/ksql/commit/ed50b05ff7789df2afc7e651b8a3c61ffeefed5a))
+* Fixes a timestamp bug with Scalable Push Queries ([#8377](https://github.com/confluentinc/ksql/pull/8377)) ([507bdb5](https://github.com/confluentinc/ksql/commit/507bdb5208697473206ccf278e9bc17c8ee1346d))
+* return error message if table functions are used inside a CASE ([#8327](https://github.com/confluentinc/ksql/pull/8327)) ([f0b9e96](https://github.com/confluentinc/ksql/commit/f0b9e962d23ef8ae225e50bd5c7012158428dd2e))
+* upgrade Netty for 5.5.x ([#8389](https://github.com/confluentinc/ksql/pull/8389)) ([e95ff38](https://github.com/confluentinc/ksql/commit/e95ff381b2ea57e4248f3ad39ed67b97fa1ad833))
+* Makes PullQueryRoutingFunctionalTest  more reliable and adds back test ([#8351](https://github.com/confluentinc/ksql/pull/8351)) ([152eea8](https://github.com/confluentinc/ksql/commit/152eea8c0b711473c438d282d885c72bf95e4f1b))
+* Makes tests wait only 500ms for consumer group cleanup ([#8367](https://github.com/confluentinc/ksql/pull/8367)) ([1a2f7b2](https://github.com/confluentinc/ksql/commit/1a2f7b27a2dff93b2dff4d22b7873e80d286d451))
+* Pull Query Key Extraction Optimizations ([#8346](https://github.com/confluentinc/ksql/pull/8346)) ([6ad333e](https://github.com/confluentinc/ksql/commit/6ad333ef6d6613155a5b4848caafcbbdce6a8a58))
+* update client to fix build ([#8369](https://github.com/confluentinc/ksql/pull/8369)) ([9ad563d](https://github.com/confluentinc/ksql/commit/9ad563d3b81d2bb50d89a46059e793436f02b566))
+* port changes from [#7863](https://github.com/confluentinc/ksql/issues/7863) to 5.4.x ([#8409](https://github.com/confluentinc/ksql/pull/8409)) ([25c9ca5](https://github.com/confluentinc/ksql/commit/25c9ca5f1f59136f574b261be43eb09947af264b))
+
+
+
+### BREAKING CHANGES
+
+* When creating connectors through ksqlDB, ksqlDB no longer automatically sets the key.converter config to be the StringConverter, as ksqlDB has supported key types other than strings for many releases now.
```

Comment:
```suggestion
* When creating connectors through ksqlDB, ksqlDB no longer automatically sets the `key.converter` config to be the `StringConverter`, because ksqlDB has supported key types other than strings for many releases now.
```